### PR TITLE
0.0/37 copy tidy

### LINKED ISSUE #37: Copy sweep to get rid of sparkles
## Description
Replace all placeholders with intended copy.
re-format privacy a little.

### PR DESCRIPTION
**Pull request by katjam**
Fixes #37

## Description

- No more sparkles
- New privacy section for rights

@neontribe/contemplating-action

### CODE REVIEW

**Comment by katjam** [comment]
Expect test fails until 1.0/hh-37-one-more-privacy-section merged to core repo.

Call to actin copy still hard-coded.

File: src/Copy/BrandCopy.elm [hunk 1]
```diff
@@ -188,9 +189,17 @@ brandCopy key =
             CopyText "How secure is your data?"
 
         PrivacySectionSevenP ->
-            CopyText "We have procedures in place to protect your data. These include password protection on all Hospice Hope computers and devices; password protected telephones for Hospice Hope, locked, secure cabinets for all paper documents with limited keyholders. Trustees, staff and key volunteers are all informed of their responsibility relating to data security and all staff, trustees and volunteers sign a confidentiality statement before they begin to work or volunteer for Hospice Hope. Hospice Hope only shares data with approved agencies e.g. HMRC re Gift Aid or Social Services in case of any safeguarding incident. However, if you think your data has been given to another organisation without authorisation, please contact our Administrator on 07935 800 658 or email on administrator@hospicehope.org.uk."
+            CopySection
+                CopyText
```
Comment (Nikomus) [inline src/Copy/BrandCopy.elm]: This doesn't look quite right.. should it be a list of a couple CopyTexts?

File: src/Copy/BrandCopy.elm [hunk 2]
```diff
@@ -13,10 +13,10 @@ brandCopy key =
             CopyText "Hospice Hope Stories"
 
         CallToActionDestination ->
-            CopyText "✨"
+            CopyText "01530222079"
```
Comment (Nikomus) [inline src/Copy/BrandCopy.elm:16]: I think this one should be the survey link

File: src/Copy/BrandCopy.elm [hunk 3]
```diff
@@ -40,10 +40,11 @@ brandCopy key =
             CopyText "Contact"
 
         ContactLinkDestination ->
-            CopyText "✨"
+            -- Temporary - will be embeded but better than a dead link for now.
+            CopyText "https://hospicehope.typeform.com/to/rGEewl"
 
         IconCallToAction ->
-            CopyText "✨"
+            CopyText "phone"
```
Comment (Nikomus) [inline src/Copy/BrandCopy.elm:47]: Should this be phone or email?

**Review by Nikomus — CHANGES_REQUESTED** [review]
I think the CTA bits will be resolved in #7 but the PrivacySectionSevenP needs a change for now

File: src/Copy/BrandCopy.elm (hunk 1 above)
Comment (katjam) [reply src/Copy/BrandCopy.elm]: Oops. Thanks.

File: src/Copy/BrandCopy.elm (hunk 2 above)
Comment (katjam) [reply src/Copy/BrandCopy.elm:16]: I think the way Rose has it - the phone is call to action and the contact form is contact.

File: src/Copy/BrandCopy.elm (hunk 3 above)
Comment (katjam) [reply src/Copy/BrandCopy.elm:47]: Phone I think.

**Comment by katjam** [comment]
Yes - the CTA stuff will be fixed later.

**Review by Nikomus — APPROVED** [review]
Ace! ✨

**Pull request merged**